app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryControls.js

```diff
@@ -54,6 +55,10 @@ class ExpandedStoryControls extends Component {
           type="button"
           value={I18n.t('cancel')}
         />
+
+        {
+          disabled && <ExpandedStoryToolTip text={I18n.t('accepted_tooltip')} />
```

Comment:
@VictorKunzler This is a good point to add a test and probably the biggest reason to coverage drop.
Please, add an extra test setting the `disabled` prop to `true`.